src/util/gamma_picker.cpp

```diff
@@ -68,14 +68,20 @@ namespace lws
 
   bool gamma_picker::is_valid() const noexcept
   {
-    return CRYPTONOTE_DEFAULT_TX_SPENDABLE_AGE < rct_offsets.size();
+    return CRYPTONOTE_DEFAULT_TX_SPENDABLE_AGE < rct_offsets.size() + 1;
```

Comment:
I would `-1` on  other side of equation, with a `static_assert` here (such that macro constant must be greater than 0.